src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -58,6 +58,11 @@ export class CalcitePickListItem {
     this.calciteListItemPropsChange.emit();
   }
 
+  /**
+   * Set this to true to display a remove action that removes the item from the list.
+   */
+  @Prop({ reflect: true }) removable = false;
```

Comment:
Should we make this an optional prop so that users know they don't have to set it?

```
@Prop({ reflect: true }) removable?: boolean;
```